### PR TITLE
Cassandra: fix compilation when "--enable-debug-messages" option is used

### PR DESCRIPTION
**Pull request by IvanNardi**
Let's try adding a dedicated compilation in GitHub Actions to easily
detect this kind of errors in the future

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -23,6 +23,12 @@ jobs:
         run: ./tests/do.sh
       - name: Test UNIT
         run: ./tests/unit/unit
+      - name: Configure (with debug logs)
+        run: make distclean && env CFLAGS='-Werror' ./autogen.sh --enable-debug-messages
+      - name: Build
+        run: make all
+      - name: Test DIFF
+        run: ./tests/do.sh
```
Comment (utoni) [inline .github/workflows/build.yml]: Is it necessary to do the diff tests twice?

Comment (utoni) [reply .github/workflows/build.yml]: Save ressources whenever possible. :)

Comment (IvanNardi) [reply .github/workflows/build.yml]: No strong opinion on that... I think that if we compile some more code, we should somehow check that it doesn't crash anything, at very least. But run the entire test suite might be a bit overwhelming...
Do you prefer to remove the tests anyway?

Comment (utoni) [reply .github/workflows/build.yml]: I mean it is just a "print some pretty debug messages"-mode. There shouldn't be lot's of additional code paths to check. And `./tests/do.sh` is always sooo slow.

Comment (IvanNardi) [reply .github/workflows/build.yml]: Ok!

**Pull request merged**